index.html

```diff
@@ -11139,11 +11139,11 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p>The <sref>aria-current</sref> <a>attribute</a> is a token type. Any value not included in the list of allowed values SHOULD be treated by <a>assistive technologies</a> as if the value <code>true</code> had been provided. If the attribute is not present or its value is an empty string or <code>undefined</code>, the default value of <code>false</code> applies and the <sref>aria-current</sref> <a>state</a> MUST NOT be exposed by user agents or assistive technologies.</p>
 				<p>The <sref>aria-current</sref> attribute is used when an element within a set of related elements is visually styled to indicate it is the current item in the set. For example:</p>
 				<ul>
-					<li>A <code>page</code> token used to indicate a link within a set of pagination links, where the link is visually styled to represent the currently-displayed page.</li>
-					<li>A <code>step</code> token used to indicate a link within a step indicator for a step-based process, where the link is visually styled to represent the current step.</li>
-					<li>A <code>location</code> token used to indicate the image that is visually highlighted as the current component of a flow chart.</li>
-					<li>A <code>date</code> token used to indicate the current date within a calendar.</li>
-					<li>A <code>time</code> token used to indicate the current time within a timetable.</li>
+					<li>A <code>page</code> token used to indicate a page within a page-based process, where the element is visually styled to represent the current page.</li>
```

Comment:
Not sure I like "page-based process". It doesn't have to be a process.
I think I prefer "set of pages". It can just be in a nav in a single-page app (for example), where the author wants to show which page is currently being displayed.
```suggestion
					<li>A <code>page</code> token used to indicate a page within a set of pages, where the element is visually styled to represent the current page.</li>
```